runners/spark/pom.xml

```diff
@@ -134,6 +152,28 @@
             <forkCount>1</forkCount>
             <reuseForks>false</reuseForks>
           </configuration>
+          <executions>
+            <execution>
+              <id>runnable-on-service-tests</id>
+                 <configuration>
+                   <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
+                <parallel>none</parallel>
+                <failIfNoTests>true</failIfNoTests>
+                <dependenciesToScan>
+                  <dependency>org.apache.beam:java-sdk-all</dependency>
+                </dependenciesToScan>
+                <systemPropertyVariables>
+                  <beamTestPipelineOptions>
+                    [
+                      "--runner=org.apache.beam.runners.spark.TestSparkPipelineRunner",
```

Comment:
Works for me with --runner=org.apache.beam.runners.spark.SparkPipelineRunner
and --streaming=false will ensure it's not a streaming pipeline.